NEWS.md

```diff
@@ -61,9 +62,14 @@
 - Added a Warning notification that appears when a uvicorn process was started with a custom port, without the root-path argument. These processes will now appear in the Proxied Servers view with a warning icon (rstudio/rstudio-pro#3699)
 - Fixed mislabelling of Shiny app in Proxied Servers list (#117)
 
-#### Jupyter Extension
+#### Posit Workbench Jupyter Sessions
+
+- Jupyter sessions are now configured dynamically to find `jupyter` using the `WORKBENCH_JUPYTER_PATH` environment variable, the configured `jupyter-exe` in `jupyter.conf` and the `PATH`. This is particularly helpful when using Kubernetes or Slurm. (rstudio/rstudio-pro#3942)
+
+#### Posit Workbench Jupyter Extension
 
 - The Jupyter Notebook and JupyterLab extensions have been updated to match with the new Posit Software, PBC branding (rstudio-pro#3645)
+- Fix issue where Workbench could not install the `jupyter` extension automatically if the configured `jupyter` was a symlink to the actual install location, because `pip` could not be found. We now follow the symlink.  (rstudio/rstudio-pro#3942)
```

Comment:
This isn't a change to the jupyter extension so it should be under the "Posit Workbench Jupyter Sessions" heading